src/js/hooks/cycleCount/useToResolveTab.jsx

```diff
@@ -0,0 +1,324 @@
+    internalLocations,
+    tags,
+    catalogs,
+    negativeQuantity,
+    searchTerm,
+  } = filterParams;
+
+  const getParams = ({
+    sortingParams,
+  }) => _.omitBy({
+    // TODO: We need to display only the rows where cycle count in [INVESTIGATING, COUNTED].
+    //       This will require us to be able to filter on the cycle count status, not just the cycle
+    //       count request status! So we need to be able to support fetching by a list of values,
+    //       or we need the get candidate API to internally fetch both.
+    status: CycleCountStatus.CREATED,
+    offset: `${offset}`,
+    max: `${pageSize}`,
+    ...sortingParams,
+    ...filterParams,
+    searchTerm,
+    facility: currentLocation?.id,
+    dateLastCount,
+    categories: categories?.map?.(({ id }) => id),
+    internalLocations: internalLocations?.map?.(({ name }) => name),
+    tags: tags?.map?.(({ id }) => id),
+    catalogs: catalogs?.map?.(({ id }) => id),
+    abcClass: [],
+    negativeQuantity,
+  }, (val) => {
+    if (typeof val === 'boolean') {
+      return !val;
+    }
+    return _.isEmpty(val);
+  });
+
+  const {
+    sortableProps,
+    sort,
+    order,
+  } = useTableSorting();
+
+  const {
+    selectRow,
+    isChecked,
+    selectHeaderCheckbox,
+    selectedCheckboxesAmount,
+    headerCheckboxProps,
+    checkedCheckboxes,
+  } = useTableCheckboxes();
+
+  const {
+    tableData,
+    loading,
+  } = useTableDataV2({
+    url: CYCLE_COUNT_CANDIDATES(currentLocation?.id),
+    errorMessageId: 'react.cycleCount.table.errorMessage.label',
+    defaultErrorMessage: 'Unable to fetch products',
+    shouldFetch: filterParams.tab,
+    getParams,
+    pageSize,
+    offset,
+    sort,
+    order,
+    searchTerm,
+    filterParams,
+  });
+
+  const getProductIds = () => tableData.data.map((row) => row.product.id);
+
+  const checkboxesColumn = columnHelper.accessor('selected', {
+    header: () => (
+      <TableHeaderCell>
+        <Checkbox
+          noWrapper
+          {...headerCheckboxProps}
+          onClick={selectHeaderCheckbox(getProductIds)}
+        />
+      </TableHeaderCell>
+    ),
+    cell: ({ row }) => (
+      <TableCell className="rt-td">
+        <Checkbox
+          noWrapper
+          onChange={selectRow(row.original.product.id)}
+          value={isChecked(row.original.product.id)}
+        />
+      </TableCell>
+    ),
+    meta: {
+      getCellContext: () => ({
+        className: 'checkbox-column',
+      }),
+    },
+  });
+
+  const columns = useMemo(() => [
+    columnHelper.accessor('cycleCountRequest.cycleCount.status', {
+      header: () => (
+        <TableHeaderCell>
+          {translate('react.cycleCount.table.status.label', 'Status')}
+        </TableHeaderCell>
+      ),
+      cell: ({ getValue }) => (
+        <TableCell className="rt-td">
+          <StatusIndicator
+            variant="primary"
+            status={translate(`react.cycleCount.table.status.${getValue()}.label`, 'To Resolve')}
```

Comment:
I wasn't sure how we normally translate values like this. The possible values are defined in an enum on the backend. In message.properties it's defined like:
```
react.cycleCount.status.COUNTED.label=To Resolve
react.cycleCount.status.INVESTIGATING.label=Resolution In Progress
...
```

Reply:
https://pihemr.atlassian.net/browse/OBPIH-6931?atlOrigin=eyJpIjoiOGE1MTUwNWMxOTUyNDhhYzhhMTIzMzdiYzcxNTIyZDkiLCJwIjoiaiJ9